source/stylesheets/base/_typography.scss

```diff
@@ -0,0 +1,18 @@
+html {
+    font-family: $font-family-base;
+}
+
+h1 {
+    font-size: $font-size-xx-large;
+    margin-block-end: 0.5rem;
```

Comment:
Same note as the previous one. This is not supported across all browsers, so consider subbing this with `margin-bottom`